src/roswire/definitions/msg.py

```diff
@@ -17,7 +17,7 @@
 from .decode import is_simple
 from .. import exceptions as exc
 
-R_TYPE = r"[a-zA-Z0-9_/]+(?:\[\d*\])?"
+R_TYPE = r"[a-zA-Z0-9_/]+(?:\[(?:<=)?\d*\])?"
```

Comment:
It might be worth just separating the fixes to this file into another PR since they're not really related to the state probe directly.

Reply:
I can't test anything related to ROS2 unless I have this package modification, which is why I'm including it in all of the PRs

Reply:
Fair enough. Situations like these are always annoying to deal with (i.e., when there's some common building block that multiple PRs need). For the future, it can help to indicate in the PR description that merging a particular PR (e.g., this one) is blocked on the merging of another (e.g., the package database PR).